source/2021-11-06-the-ember-times-issue-167.md

```diff
@@ -24,14 +24,15 @@ Read more about the new changes to [ember-a11y-testing on GitHub](https://github
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Release 2.0.x of ember-select-light 🎉](https://github.com/ember-a11y/ember-select-light/releases/tag/v2.0.0)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Ava Wroten (@hergaiety)](https://www.wroten.me/) released 2.0.0 of [ember-select-light](https://github.com/ember-a11y/ember-select-light/) moving the addon into the [ember-a11y org](https://github.com/ember-a11y) (join us on [discord in #dev-ember-a11y](https://discord.gg/xBzFBMYv)) and making it **Octane ready**.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+**ember-select-light is a simple and accessible `<select>` dropdown component** ✨. With a simple API you can pass in a flat or nested array of options or yield out to your own manually handled options template code. This component pairs nicely with the latest [ember-component-patterns for Select Elements](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/select-element).
```

Comment:
Right now, we're having issues with Travis CI not running fast enough. Can you run `alex` on this file? I think it may ask you to use another word besides "simple" (you can make an exception if you prefer).

```bash
# If you don't want to install alex globally, I think you can try:
npx alex source/2021-11-06-the-ember-times-issue-167.md
```

Reply:
To help with parsing the sentences, maybe:

```suggestion
**ember-select-light is a simple and accessible `<select>` dropdown component** ✨. With a simple API you can pass in a flat or nested array of options, or yield out to your template code that handles options. The component also pairs nicely with the latest [ember-component-patterns for Select Elements](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/select-element).
```